docs/docs/cmd/request.md

```diff
@@ -0,0 +1,63 @@
+# request
+
+Executes the specified web request using CLI for Microsoft 365
+
+## Usage
+
+```sh
+m365 request [options]
+```
+
+## Options
+
+`-u, --url <url>`
+: The request URL. 
+
+`-m, --method [method]`
+: The HTTP request method. Accepted values are `get, post, put, patch, delete, head, options`. The default value is `get`.
+
+`-r, --resource [resource]`
+: The resource url for which the CLI should acquire a token from AAD in order to access 
+the service.
+
+`-b, --body [body]`
+: The request body. Optionally use `@example.json` to load the body from a file. 
+
+`-a, --accept [accept]`
+: A convenience option to set the `accept` header of the request. The default to `application/json`. 
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+The request will be issued as bare as possible, meaning you are responsible for all request information, such as headers, method and body. The only exception is the `authorization` header. By default, the command will try to retrieve a valid token for the API you are executing a request against based on the `url` option. 
```

Comment:
We'll also apply compression and throttling handling as it's a part of our request execution.